### ISSUE TITLE
AWS Snapshot Storage Tiering

### ISSUE THREAD

**Issue by scoleri** [cloud-custodian/cloud-custodian#7022]
### Describe the feature

New feature in AWS snapshots are storage tiering for cost optimization.  You can move snapshots to archival and save money.  Would love this in Cloud Custodian.

https://aws.amazon.com/about-aws/whats-new/2021/11/retention-ebs-snapshots-75-amazon-archive-tier/
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/working-with-snapshot-archiving.html

**Comment by kapilt** [comment]
draft pr up

**Comment by kapilt** [comment]
@scoleri do you have an example scenario where this would save money.. when you archive you convert an incremental snapshot to a full archive. https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/archiving-guidelines.html

**Comment by scoleri** [comment]
This is a new aws feature for snapshots.  Tiered.

https://aws.amazon.com/about-aws/whats-new/2021/11/retention-ebs-snapshots-75-amazon-archive-tier/

**Comment by kapilt** [comment]
yeah, but knowing if you can use it is more nuanced then that, per my previous archiving-guidelines link. my concern is that simply exposing it without an ability to understand if its actually a cost benefit can lead to negative outcomes, hence the desire to understand worked examples/use cases as would be expressed by custodian.

**Comment by kapilt** [comment]
ie the formula for cost benefit is actually a bit complicated, and the actualization of incremental into a full snapshot can increases costs if used naively.. https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/archiving-guidelines.html#archive-guidelines

**Comment by leungtimothy** [comment]
> @scoleri do you have an example scenario where this would save money.. when you archive you convert an incremental snapshot to a full archive. https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/archiving-guidelines.html

hey @kapilt, we'd find this useful as we are looking to snapshot + delete a large amount of ebs volumes using cloud custodian. We need to retain snapshots for compliance reasons but have no intention to pull them out of the Archive state.   The cost difference is pretty significant (75%).

**Comment by kapilt** [comment]
we can add it in, with a big warning, that it may increase costs if used naively.
